config/Wasteland/ChestLoot.cfg

```diff
@@ -163,9 +163,7 @@
         minecraft:pumpkin_pie,1,1
         minecraft:red_flower,1,1
         minecraft:iron_ingot,2,1
-        minecraft:compass,1,1
```

Comment:
we don't need that?

Reply:
with journey map its pretty useless. even if it is a crafting recipe item its easy to make.